src/components/AuthCheck.tsx

```diff
@@ -1,20 +1,40 @@
-import { useQuery } from "@tanstack/react-query"
+import { useMutation, useQuery } from "@tanstack/react-query"
 import { Redirect } from "wouter"
 
 import { getUser } from "../utils/query-user"
 import { FullPageLoader } from "./Loader"
+import { loginToSite } from "../utils/switch-site"
+import { useEffect } from "react"
+import { ThemeKey } from "../types/theme"
+import { getCurrentTheme } from "../utils/current-theme"
```

Comment:
these aren't used. I'd suggest you also add ESLint to catch unused imports. It could be another PR not to make this one too big unnecessarily.